samples/http-image-cloudevents/test_image_sample.py

```diff
@@ -80,6 +83,44 @@ def test_create_structured_image():
     assert restore_image.size == image_expected_shape
 
 
+def test_server_structured(client):
```

Comment:
Function name could be clearer imo. Something like:

test subject verb object

`test_server_send_structured_cloudevent`

Something that would make it readable if listing all the functions would be nice.